src/components/SelectionList/CheckboxListItem.js

```diff
@@ -0,0 +1,79 @@
+import Avatar from '../Avatar';
+import OfflineWithFeedback from '../OfflineWithFeedback';
+import useLocalize from '../../hooks/useLocalize';
+
+function CheckboxListItem({item, isFocused = false, onSelectRow, onDismissError = () => {}}) {
+    const {translate} = useLocalize();
+    const hasError = !_.isEmpty(item.errors);
+
+    return (
+        <OfflineWithFeedback
+            onClose={() => onDismissError(item)}
+            pendingAction={item.pendingAction}
+            errors={item.errors}
+            errorRowStyles={styles.ph5}
+        >
+            <PressableWithFeedback
+                style={[styles.peopleRow, styles.userSelectNone, isFocused && styles.sidebarLinkActive, hasError && styles.borderColorDanger]}
+                onPress={() => onSelectRow(item)}
+                disabled={item.isDisabled}
+                accessibilityLabel={item.text}
+                accessibilityRole="checkbox"
+                accessibilityState={{checked: item.isSelected}}
+                hoverDimmingValue={1}
+                hoverStyle={styles.hoveredComponentBG}
+                focusStyle={styles.hoveredComponentBG}
+            >
+                <Checkbox
+                    accessibilityLabel={item.text}
+                    disabled={item.isDisabled}
+                    isChecked={item.isSelected}
+                    onPress={() => onSelectRow(item)}
+                    style={item.isDisabled ? styles.buttonOpacityDisabled : {}}
+                />
+                {item.avatar && (
+                    <Avatar
+                        containerStyles={styles.pl5}
+                        source={item.avatar.source}
+                        name={item.avatar.name}
+                        type={item.avatar.type}
+                    />
+                )}
+                <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStart, styles.pl3, styles.optionRow]}>
+                    <Text
+                        style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, item.isSelected && styles.sidebarLinkTextBold]}
+                        numberOfLines={1}
+                    >
+                        {item.text}
+                    </Text>
+                    {Boolean(item.alternateText) && (
+                        <Text
+                            style={[isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting]}
+                            numberOfLines={1}
+                        >
+                            {item.alternateText}
+                        </Text>
+                    )}
+                </View>
+                {item.isAdmin && (
+                    <View style={[styles.badge, styles.peopleBadge]}>
+                        <Text style={styles.peopleBadgeText}>{translate('common.admin')}</Text>
+                    </View>
+                )}
```

Comment:
This is something relevant to MembersPage, why this is part of the Component Item?

Reply:
For the moment, yes, but maybe in the future we could have more usages. 

Or we could instead of looking for `item.isAdmin`, make it more generic like `item.rightBadge` (or any other name) so other lists could add components in that spot if needed.

But I know design team is having a look at all lists interactions, so I didn't want to overthink this, as many things can change. So I kept it simple.

Reply:
@cristipaval Your thoughts?

Reply:
I do agree that we should either give a more generic name or implement it for the MembersPage only.

Reply:
let's just rename it for now @thiagobrez. I'd merge this one faster instead of continuing to enlarge the already very big scope of it

Reply:
@cristipaval @Santhosh-Sellavel Agree, fixed ✅